### ISSUE TITLE
WCAG 4.1.2 test 10 false positive?

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#223]
[Test document 15](https://github.com/veraPDF/veraPDF-wcag-algs/files/9111663/test-document-15.pdf) fails WCAG 4.1.2 test 10 (_Heading is incorrectly tagged as a paragraph_) in one place, with the line that reads "**5. Lettertypen**". Visually, this appears to be a valid heading at the same level as the other headings in the document.

[Test document 14](https://github.com/veraPDF/veraPDF-wcag-algs/files/9112025/test-document-14.pdf) fails WCAG 4.1.2 test 10 in seven places. The first three  of these (on pages 11, 34, 38) appear to be valid numbered headings.

Can you check if these are false positives? Or is there really something wrong with the tagging of these particular headings?

**Comment by irinamavrina** [comment]
Hi @jzuidweg 
We have checked these documents, and it appears that we really do recognize these places as headings, which is correct, as they look like ones, but in the document these places are tagged as paragraphs. So we have here headings that are incorrectly tagged as paragraphs and that is why the test fails.

**Comment by jzuidweg** [comment]
Thanks for sorting this out, Irina. I think we can close this issue then, as this does not seem to be a false positive.
